src/averageAges.js

```diff
@@ -37,7 +42,14 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const womanWithChild = withChildren
+    ? people.filter(allPeople => people
```

Comment:
```suggestion
    ? people.filter(person => people
```
change to more meaningful name